### ISSUE TITLE
Show picture profile on account list

### ISSUE THREAD

**Issue by Vucumpra** [thunderbird/thunderbird-android#8238]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

8.0b1

### Problem you are trying to solve

I have three Gmail accounts, all displayed as [GM] :/

### Suggested solution

Show picture from address book

### Screenshots / Drawings / Technical details

![Screenshot_20241004-104245](https://github.com/user-attachments/assets/f614ea26-b91d-4156-98c7-21d50a02ec4b)

**Comment by MillionsToOne** [comment]
Yeah. If no picture is set, maybe use the first letters of the names instead of the first 2 letters of the domain.

**Comment by MrEAlderson** [comment]
![Screenshot_20241005_204648_Thunderbird Beta](https://github.com/user-attachments/assets/ec59a27c-8097-47ed-b42d-70271c78bec1)
It is no fun guessing which account is which - K9 at least had a drop down picker. The account picker is at its current stage unusable

**Comment by Vucumpra** [comment]
Yes, in order, select picture from : 
1- Personal address book
2- OR Remote account
3- OR firsts 2 letters of ID

**Comment by kewisch** [comment]
We originally planned to add a picture but went with this for time reasons. The picture could come from the contacts, or potentially setting one. 

The letters from the domain were a good compromise since in many cases folks will have eg a personal email on gmail, a work email, and maybe another email from a different provider. We decided against using parts of the name, because it is more likely to be the same if used by the same person. Having multiple separate email accounts at the same domain might be more of an edge case.

That is not to say it should stay this way. If someone wants to work on this it would be a great project for someone with past Android development experience.

**Comment by kewisch** [comment]
I am happy to share the original mockups if needed

**Comment by marcRDZ** [comment]
> I am happy to share the original mockups if needed

Hi @kewisch can you show or send me them? I'd like to work on this awesome feature. :)

**Comment by Julien-Delavisse** [comment]
Hello, I wonder why users prefer to see an image associated with an account, rather than ... the email address.
Can you tell us a little more about the choice to switch from the old design (e-mail address) to the new one (two letters ...)?
It may be “prettier”, but the user experience is much less pleasant.
Do you have several accounts and do you find it pleasant to guess which account is behind “AB” or “CD”? Not to mention the duplicates...
Imagine if on a messaging system like Teams/Slask, people's names were summarized in two characters. Yet that's what Thunderbird chooses?

To try to find a solution, can you make a test version where the circle would be replaced by at least the username?

Or maybe make the top left button toggle between two modes (which is useless today? ):
- the current display with the list of accounts and folders
- a display with the list of accounts and full e-mail addresses

This would give all users the best of both worlds.

I sincerely believe that this issue should be reopened : https://github.com/thunderbird/thunderbird-android/issues/8447

**Comment by MillionsToOne** [comment]
People with multiple accounts from same email service, for example gmail, had gm on all of the icons.

**Comment by Julien-Delavisse** [comment]
> People with multiple accounts from same email service, for example gmail, had gm on all of the icons.

I didn't understand your comment. Yes, having a profile picture is a solution to the problem we have. But much less obvious than seeing the e-mail address directly. Would you prefer an image? It works for e-mails from individuals, but much less so for generic corporate e-mails, for example. (contact mail, support mail, etc.).

**Comment by MrEAlderson** [comment]
All I want is the dropdown from K-9. Greatly doubt that anyone clicks on the mail just to move to the next account in order, although that could stay if only 2 accounts were added

**Comment by kewisch** [comment]
Please follow the work in #8812, you'll see some info in our blog post as well: https://blog.thunderbird.net/2025/03/thunderbird-for-android-january-february-2025-progress-report/

Closing this in favor of 8812
